Library/Homebrew/cask/url.rb

```diff
@@ -169,6 +172,11 @@ def url(uri, &block)
       # This allows calling DSL methods from inside a `url` block.
       #
       # @api public
+      sig {
+        override
+          .params(method: Symbol, args: T.untyped, block: T.nilable(T.proc.returns(T.untyped)))
+          .returns(T.untyped)
```

Comment:
```suggestion
          .returns(T.anything)
```
Learned about it recently and we could probably use it more

Reply:
Hmm, I agree with this generally, but in this case I'm matching the sig of the super definition: https://github.com/sorbet/sorbet/blob/23b1dbd68a04cf2bfe2b6f3d05e6e5c340b0fce4/rbi/core/basic_object.rbi#L301-L302

(well, mostly, i think there's a bug there in not specifying a block param)

I think returning the top type (T.anything) here violates Liskov substitution, though it's probably moot (`method_missing` is unlikely to be invoked statically, and neither option will be checked at runtime).